doc/configuration.rst

```diff
@@ -598,6 +598,33 @@ examples into a single mini-gallery, e.g.:
 For such a mini-gallery, specifying a custom heading message is recommended
 because the default message is vague: "Examples of one of multiple objects".
 
+Create mini-galleries using filepaths
+"""""""""""""""""""""""""""""""""""""
+Sometimes you may want to explicitly create a mini-gallery for files that may be 
+in the same location but not have functions in common, for example a set of 
+tutorials. The mini-gallery directive supports passing in a string path and 
+a glob-style string to a set of files. 
+
+.. code-block:: rst
+
+    .. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+
+
+.. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+   :add-heading: Mini-gallery using paths
+   :heading-level: -
+
+The order of the minigallery file based entries can be set using the 
+``minigallery_sort_order`` configuration key. This configuration key supports
+any callable that is a valid *key* argument to ``sorted``::
+
+    sphinx_gallery_conf = {
+    ...,
+    'minigallery_sort_order' : lambda x: return len(x)%4,
+    ... 
+    }
```

Comment:
Instead of a trivial example here, could you make one that actually does something interesting related to the example above with `numpy.exp ../examples/...`? I think it will be easier for people to understand the use case